docs/how_to_guides/streaming.ipynb

```diff
@@ -37,9 +37,59 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
+   "execution_count": 2,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/zaydsimjee/workspace/guardrails/guardrails/validators/__init__.py:50: FutureWarning: \n",
+      "    Importing validators from `guardrails.validators` is deprecated.\n",
+      "    All validators are now available in the Guardrails Hub. Please install\n",
+      "    and import them from the hub instead. All validators will be\n",
+      "    removed from this module in the next major release.\n",
+      "\n",
+      "    Install with: `guardrails hub install hub://<namespace>/<validator_name>`\n",
+      "    Import as: from guardrails.hub import `ValidatorName`\n",
+      "    \n",
+      "  warn(\n",
+      "\n",
+      "/Users/zaydsimjee/workspace/guardrails/guardrails/validator_base.py:397: FutureWarning: Accessing `LowerCase` using\n",
+      "`from guardrails.validators import LowerCase` is deprecated and\n",
```

Comment:
nit: can you updated to `from guardrails.hub import LoweCase`? Same for other validator imports below.